detection_rules/rule.py

```diff
@@ -361,6 +361,47 @@ def process_note_plugins():
 
         return obj
 
+    @validates_schema
+    def validate_bbr(self, value: dict, **kwargs):
+        def convert_time_span(span: str) -> int:
+            """Convert time span in datemath to value in milliseconds."""
+            amount = int("".join(char for char in span if char.isdigit()))
+            unit = eql.ast.TimeUnit("".join(char for char in span if char.isalpha()))
+            return eql.ast.TimeRange(amount, unit).as_milliseconds()
+
+        def skip_validate_bbr() -> bool:
+            return os.environ.get('DR_BYPASS_BBR_LOOKBACK_VALIDATION') is not None
+
+        def validate_time_defaults(str_time) -> bool:
+            """Validate that the time is at least now-119m and at least 60m respectively."""
+            try:
+                if "now" in str_time:
+                    str_time = str_time.replace("now-", "")
+                    time = convert_time_span(str_time)
+                    # if time is less than 119m as milliseconds
+                    if time < 119 * 60 * 1000:
+                        return False
+                else:
+                    time = convert_time_span(str_time)
+                    # if time is less than 60m as milliseconds
+                    if time < 60 * 60 * 1000:
```

Comment:
what is the premise here for it to be only 60m v 119m?

Reply:
The 60m is the default interval, and the 119m is from the `from` validation of `now-119m`. These values were chosen to maintain the desired ratios for the `eql` unit tests as a building block rule can also be of the type `eql` while maintaining the desired "run once every hour" default we wanted to enforce on building block rules.